loops-and-arrays.js

```diff
@@ -23,6 +32,20 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 
 const numbers = [2, 4, 5, 29, 38];
 
+console.log("Voor het script opdracht 2:");
+console.log(numbers);
+
+for (let i = 0; i < numbers.length; i++) {
+    if (numbers[i] % 2 === 0) {
+        numbers[i] = numbers[i] * 2;
+    } else {
```

Comment:
1. na de "}" mag een enter, zodat 'else' op de volgende regel start.
2. Novi gebruikt hier een 'if-statement' in plaats van een 'else-statement'. Zelf had ik ook 'else', gezien het even of oneven is. Ik kan je dus niet aangeven, waarom je hier een if-statement had moeten plaatsen.